src/work-allocation-2/components/task-field/task-field.component.scss

```diff
@@ -0,0 +1,7 @@
+.xui-image-field__container {
```

Comment:
Again, this file should not exist. This should be present in file called task-field.component.scss